<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIAskEverything</class>
 <widget class="QMainWindow" name="AIAskEverything">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AIAskEverything</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QTextEdit" name="te_question"/>
    </item>
    <item>
     <widget class="QPushButton" name="b_ask">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="te_answer"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="Balls"/>
    </item>
    <item>
     <widget class="QPushButton" name="Check">
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
